client/src/components/brushableHistogram/index.js

```diff
@@ -319,18 +319,31 @@ class HistogramBrush extends React.PureComponent {
     /* Remove everything */
     svg.selectAll("*").remove();
 
+    var yMax = d3.max(bins, function (d) { return d.length });
+    var yMin = d3.min(bins, function (d) { return d.length });
+
+    var colorScale = d3.scaleSequential(d3.interpolateViridis)
```

Comment:
Thanks a ton for submitting this PR, which I believe is cellxgene's first externally developed feature! 

Instead of Viridis (which was previously used), cellxgene makes use of `import { interpolateCool } from "d3-scale-chromatic";` to be uniform with the color range and palette of the categories in the lefthand sidebar's sampling from `interpolateRainbow`.

You can see this in action in `src/components/continuousLegend/index.js`



(Thanks as well to @liaprins-czi for documenting features in issues which facilitates community contribution!)

Reply:
Thanks, @colinmegill for the warm welcome.  I've been looking for an interesting open-source project to contribute to.  With this one, I can use the skills from my first career when I was a biochemist instead of a software dev.
    I'll take a look at implementing this feature with interpolateCool instead of Viridis.

Reply:
:) Wonderful. Glad we're a fit! Will look for your PR.